.github/workflows/.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
```

Comment:
- [ ] Good job setting up the work flow, but the `.hintrc` file should be in the root of your repository. Kidly add it there as its placed on the linter file requirements 👍